### PR TITLE
Update readme to cite Science paper

### PR DESCRIPTION
**Pull request by hyanwong**

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Might be nice to add a [CITATION.cff](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) also

**Comment by hyanwong** [comment]
> Might be nice to add a [CITATION.cff](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) also

Do we want to cite the science paper here? I'm not sure we do, and it looks like CFF files usually create `@software` BibTex entries, rather than `@article` ones. If so, what do we provide as the CFF?

**Comment by jeromekelleher** [comment]
There's a [preferred-citation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#citing-something-other-than-software) field for this

**Comment by hyanwong** [comment]
> There's a [preferred-citation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#citing-something-other-than-software) field for this

Oh, brilliant, thanks. I don't know why I couldn't find that on initial browsing of CFF tutorials. I'll stick the science paper in then.

**Comment by hyanwong** [comment]
Unless we want to cite the Zenodo software release, I guess (10.5281/zenodo.5168040)

**Comment by hyanwong** [comment]
Done. Added both the Zenodo dot and (preferred citation) the science paper. Should be OK to merge I hope. Not sure how I test the CFF.

**Comment by awohns** [comment]
In keeping with standard practice in the field, the Science paper should
definitely be the preferred citation I think, zenodo citations really
aren’t easy for people to find or for indexing websites to track.

**Comment by jeromekelleher** [comment]
I wouldn't bother with the Zenodo citation, I think that'll just confuse people. We want people to cite the paper. Plus the zenodo releases only make sense when applied to particular version, and we're not going to keep these up to date as tsdate gets updated and released.

**Comment by hyanwong** [comment]
True. Removed in the latest push.

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  family-names: Jeffery
+  orcid: "https://orcid.org/0000-0002-1982-6801"
+title: "Tsdate"
+version: 0.1.4
+date-released: 2021-06-30
+url: "https://github.com/tskit-dev/tsdate"
+license: MIT
+preferred-citation:
+  type: article
+  authors:
```
[the remark this answers is not in the document]
Comment (hyanwong) [reply CITATION.cff:20]: Oh, well spotted. Sorry. Corrected in latest push

**Pull request merged**